flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala

```diff
@@ -54,6 +54,9 @@ case class Project(projectList: Seq[NamedExpression], child: LogicalNode) extend
             case expr if !expr.valid => u
             case c @ Cast(ne: NamedExpression, tp) => Alias(c, s"${ne.name}-$tp")
             case gcf: GetCompositeField => Alias(gcf, gcf.aliasName().getOrElse(s"_c$i"))
+            case over: OverCall if null != over.aggAlias =>
```

Comment:
Currently `.select('c, 'b.count over 'w as 'mycount)` would be translated into `OverCall(Count(b), w, mycount, _)`. If we remove `as` from `OverCall`, it will be wrapped in a regular `Alias` expression: `Alias(OverCall(Count(b), w, _), mycount)`.

The syntax would be the same, but the internal representation would use the existing `Alias` expression, just like any other expression that is renamed.